USFMToolsSharp.Renderers.Docx/DocxRenderer.cs

```diff
@@ -194,17 +194,15 @@ private void RenderMarker(Marker input, StyleConfig styles, XWPFParagraph parent
                     break;
             }
         }
-        private void RenderFootnotes()
+        private void RenderFootnotes(StyleConfig markerStyle)
```

Comment:
Do the footnotes want to follow a style from the parent document?

Reply:
No they should have a new font size. I updated the styleconfig in the renderfootnotes method.